### DISCUSSION TITLE
Before and after modifiers not working  please help

### DISCUSSION THREAD

**Discussion by WebLearnma** [tailwindlabs/tailwindcss#9356] [Help]
Syntax error: G:\nextprojects\disneynext\styles\globals.css The `before:` class does not exist. If `before:` is a custom class, make sure it is defined within a `@layer` directive.

**Comment by wongjn** [comment]
How are you attempting to use it? What tailwind version are you using? Could we see your CSS file? Could you replicate your issue on [Tailwind Play](https://play.tailwindcss.com)?

**Reply by WebLearnma** [reply]
thank you

**Comment by adamwathan** [comment]
My guess is you are trying to use `@apply` and you have a space in your code where there shouldn’t be one, like:

`@apply before: block`

**Reply by WebLearnma** [reply]
thank you
